lvsfunc/deblock.py

```diff
@@ -258,41 +252,137 @@ def _get_strength_clip(clip: vs.VideoNode, strength: SupportsFloat) -> vs.VideoN
     else:
         clip_rgb = kernel.resample(clip_32, vs.RGBS, matrix_in=targ_matrix).std.Limiter()
 
-    mod_w, mod_h = clip_rgb.width % 8, clip_rgb.height % 8
+    if overlap is None:
+        overlap_w = overlap_h = 0
```

Comment:
Could we give this a default? Maybe auto-calculated based on the dimensions?

Reply:
This should only be set if there is an issue with tiled processing

Reply:
You always benefit from a slight overlap when tiling, though.